cheroot/errors.py

```diff
@@ -66,6 +72,7 @@ def plat_specific_errors(*errnames):
 
 
 acceptable_sock_shutdown_error_codes = {
+    errno.EBADF,
```

Comment:
Can we have something like

```suggestion
_bad_fd_errors = {errno.EBADF}
if IS_WINDOWS:
    _bad_fd_errors(socket.WSAENOTSOCK)

acceptable_sock_shutdown_error_codes = {
    *_bad_fd_errors,
```

? Perhaps, with that fallback you're trying to add to `makefile`. We could just have that here.